packages/ui/components/ui/tx/actions-views/delegator-vote.tsx

```diff
@@ -116,7 +117,17 @@ const VoteBodyDetails = ({ body }: { body?: DelegatorVoteBody }) => {
       )}
       {!!body?.unbondedAmount && (
         <ActionDetails.Row label='Voting power'>
-          <ValueViewComponent view={umValueView(body.unbondedAmount)} />
+          <Density compact>
+            <div className='ml-4'>
+              <ValueViewComponent
```

Comment:
each `ValueViewComponent` will similarly need to set the `showValue` prop to `true`, except the [swap](https://github.com/prax-wallet/prax/pull/241/files#diff-65e3447520ce9e2e65142d9db26341252ce7edc696056c012ee1cf00cb46875b)